src/OpenTelemetry.Instrumentation.Runtime/description.md

```diff
@@ -0,0 +1,198 @@
+
+[JitInfo.GetCompilationTime](https://docs.microsoft.com/dotnet/api/system.runtime.jitinfo.getcompilationtime?view=net-6.0#system-runtime-jitinfo-getcompilationtime(system-boolean)):
+Gets the amount of time the JIT Compiler has spent compiling methods.
+The scope of this value is global.
+
+## Threading related metrics
+
+The metrics in this section can be enabled by setting the
+`RuntimeMetricsOptions.IsThreadingEnabled` switch.
+
+These metrics are only available for NETCOREAPP3_1_OR_GREATER.
+
+| Name                                                        | Description                          | Units       | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|-------------------------------------------------------------|--------------------------------------|-------------|-------------------|------------|------------------|------------------|
+| process.runtime.dotnet.**monitor.lock.contention.count**    | Monitor Lock Contention Count        | `{times}`   | ObservableCounter | `Int64`    |                  |                  |
+| process.runtime.dotnet.**threadpool.thread.count**          | ThreadPool Thread Count              | `{threads}` | ObservableGauge   | `Int32`    |                  |                  |
+| process.runtime.dotnet.**threadpool.completed.items.count** | ThreadPool Completed Work Item Count | `{items}`   | ObservableCounter | `Int64`    |                  |                  |
+| process.runtime.dotnet.**threadpool.queue.length**          | ThreadPool Queue Length              | `{items}`   | ObservableGauge   | `Int64`    |                  |                  |
+| process.runtime.dotnet.**active.timer.count**               | Number of Active Timers              | `{timers}`  | ObservableGauge   | `Int64`    |                  |                  |
+
+- [Monitor.LockContentionCount](https://docs.microsoft.com/dotnet/api/system.threading.monitor.lockcontentioncount?view=netcore-3.1):
+  Gets the number of times there was contention when trying to take the monitor's
+  lock.
+- [ThreadPool.ThreadCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.threadcount?view=netcore-3.1):
+  Gets the number of thread pool threads that currently exist.
+- [ThreadPool.CompletedWorkItemCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.completedworkitemcount?view=netcore-3.1):
+  Gets the number of work items that have been processed so far.
+- [ThreadPool.PendingWorkItemCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.pendingworkitemcount?view=netcore-3.1):
+  Gets the number of work items that are currently queued to be processed.
+- [Timer.ActiveCount](https://docs.microsoft.com/dotnet/api/system.threading.timer.activecount?view=netcore-3.1):
+  Gets the number of timers that are currently active. An active timer is registered
+  to tick at some point in the future, and has not yet been canceled.
+
+## Process related metrics
```

Comment:
I have a general question - do we want these to be in the `OpenTelemetry.Instrumentation.Runtime` package, or we want them to be in a different package (and if yes, what should be the name of that package)?

Reply:
I'd vote separate package and OpenTelemetry.Instrumentation.Process as a potential name?

Reply:
Created a PR to remove it: #446

Reply:
I have the CPU metrics percentage collector...  and will also push that to new namespace... https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/437  thx @reyang  for ping about info of new namespace...